canonical-web-and-design/maas-ui
# refactor(ui): move network fields to component

## Done

- Make a reusable component for the network fields.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- View the network details for a machine.
- Click 'Add interface'.
- Enter some values for the network fields and submit.
- The interface should be created with the correct network values.

## Fixes

Fixes: #2203.
